DanielaaER.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Daniela Espinosa",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Engineering Student",
+  "avatar": "https://github.com/DanielaaER.png",
+  
```

Comment:
HI @DanielaaER! While the JSON is valid, you can consider removing this space as it is not needed.